services/mouse.md

```diff
@@ -0,0 +1,37 @@
+# Mouse
+
+    identifier: 0x1885dc1c
+
+Control a device that acts as a mouse.
+
+## Commands
+
+    enum Button : u8 {
+        right = 0x01
+        middle = 0x04
+        left = 0x02
+    }
+    enum ButtonEvent : u8 {
+        up = 0x01
+        down = 0x02
+        click = 0x03
+        double_click = 0x04
```

Comment:
I think we use `DoubleClick` casing elsewhere

Reply:
I don't think USB mouse is generating click or double click events, is it?

Reply:
I am thinking that the device generates those by sending up/down events with the correct timing. Relying on the jacdac bus packets is probably too brittle. Sending a click is very common.

Reply:
sounds good!